From 234fe1bb4c53def8b41f55105d69caeda3b51312 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Thu, 25 Jun 2020 16:31:59 +0200
Subject: [PATCH 01/16] profiles/wifi.yml: add profile for OpenWrt HEAD wifi

Selecting this profile will override the current wifi stack with the v5.7
version from OpenWrt HEAD.

Signed-off-by: John Crispin <john@phrozen.org>
---
 profiles/wifi.yml | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 profiles/wifi.yml

diff --git a/profiles/wifi.yml b/profiles/wifi.yml
new file mode 100644
index 0000000000..e3cbd04b0f
--- /dev/null
+++ b/profiles/wifi.yml
@@ -0,0 +1,13 @@
+---
+description: Add the wifi support from OpenWrt HEAD
+feeds:
+  - name: wifi
+    path: ../../feeds/wifi-trunk
+packages:
+  - ath10k-firmware-qca4019-ct-htt
+  - ath10k-firmware-qca9888-ct-htt
+  - ath10k-firmware-qca9984-ct-htt
+diffconfig: |
+  # CONFIG_PACKAGE_ath10k-firmware-qca4019-ct is not set
+  # CONFIG_PACKAGE_ath10k-firmware-qca9888-ct is not set
+  # CONFIG_PACKAGE_ath10k-firmware-qca9984-ct is not set
-- 
2.25.1

